assets/lambda/ImageOptimization.ts

```diff
@@ -0,0 +1,139 @@
+// Taken from: https://github.com/sladg/nextjs-lambda/blob/master/lib/standalone/image-handler.ts
+// There are other open source MIT libraries we can pick, but this seems the most straightforward
+
+import { GetObjectCommand, S3Client } from '@aws-sdk/client-s3'
+import type { APIGatewayProxyEventV2, APIGatewayProxyStructuredResultV2 } from 'aws-lambda'
+import { IncomingMessage, ServerResponse } from 'http'
+import { defaultConfig, NextConfigComplete } from 'next/dist/server/config-shared'
+import { imageOptimizer as nextImageOptimizer, ImageOptimizerCache } from 'next/dist/server/image-optimizer'
+import { NextUrlWithParsedQuery } from 'next/dist/server/request-meta'
+import { ImageConfigComplete, ImageConfig } from 'next/dist/shared/lib/image-config'
+import { Writable } from 'node:stream'
+import https from 'node:https'
+
+const sourceBucket = process.env.S3_SOURCE_BUCKET ?? undefined
+
+/**
+ * The imageConfig object is stringified and passthrough the lambda as an environment variable
```

Comment:
```suggestion
 * The imageConfig object is stringified and passed to lambda as an environment variable
```

Reply:
we could copy the required-server-files.json manifest to the bundle and read it the same as the server does
not a big deal

Reply:
Interesting, I missed how this was done w/ the server lambda 👍

Reply:
Good call, I didn't realized require-server-files.json had all the configs. Was looking at images-manifest.json but that wasn't useful b/c the regex was already transformed.